Sources/Css/Render.swift

```diff
@@ -33,7 +33,7 @@ func render(attribute: CssSelector.Attribute) -> String {
   }
 }
 
-func renderSelector(_ config: Config, _ selector: CssSelector) -> String {
+public func renderSelector(_ selector: CssSelector) -> String {
```

Comment:
I only wanted this public so that I could create a `class` helper for HTML attributes that took an array of `CssSelector`s, but also saw that `config` wasn't even being used.